packages/components/app/styles/components/copy/button.scss

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// COPY > BUTTON
+//
+
+// mostly the hds-button but with a few style tweaks
+// so the CSS is scoped to the component class (we're not changing any of the other class names)
+.hds-copy-button {
+  &.hds-button--color-secondary {
+    .hds-button__icon {
+      color: var(--token-color-foreground-action);
+    }
+
+    &.hds-copy-button--success {
+      .hds-button__icon {
+        color: var(--token-color-foreground-success);
```

Comment:
should this be `--token-color-foreground-success-on-surface`?

cc @andgen404

Reply:
Good catch, yeah I use `--token-color-foreground-success-on-surface` for the clipboard-checked icon